gitlab-kas.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: gitlab-kas
+  version: 14.5.0
+  epoch: 0
+  description: GitLab Kas is a component installed together with GitLab. It is required to manage the GitLab agent for Kubernetes.
+  copyright:
+    - license: MIT
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: fetch
+    with:
+      # https://gitlab.com/gitlab-org/build/CNG/-/tree/master/gitlab-kas
+      uri: https://gitlab.com/gitlab-org/cluster-integration/gitlab-agent/-/archive/v${{package.version}}/gitlab-agent-v${{package.version}}.tar.gz
+      expected-sha256: 52270e37de7f0c002e13a3f1e65cb7d0debea0fc99fcb97171b02457dbb2e0d5
+
+  - uses: go/build
+    with:
+      packages: ./cmd/kas
+      output: kas
+      ldflags: "-w -X gitlab.com/gitlab-org/cluster-integration/gitlab-agent/v14/cmd.Version=v${{package.version}} -X gitlab.com/gitlab-org/cluster-integration/gitlab-agent/v14/cmd.Commit=v${{package.version}} -X gitlab.com/gitlab-org/cluster-integration/gitlab-agent/v14/cmd.BuildTime=$(date +%F-%T)"
```

Comment:
Does this work with `date`?

Reply:
When running `kas -v` I got `kas version v14.5.0, commit: v14.5.0, built: 2023-10-10-14:02:11`